panel/pane/perspective.py

```diff
@@ -130,6 +132,41 @@ def _init_params(self):
         props['source'] = ColumnDataSource(data=self._data)
         return props
 
+    def _process_param_change(self, msg):
+        msg = super()._process_param_change(msg)
+        for param in ('columns', 'row_pivots', 'column_pivots'):
+            if msg.get(param):
+                msg[param] = [str(col) for col in msg[param]]
```

Comment:
```suggestion
        for par in ('columns', 'row_pivots', 'column_pivots'):
            if msg.get(par):
                msg[par] = [str(col) for col in msg[par]]
```
I think this is the reason for the flake error.